Remove WASM from 23.1 (#1549)
redpanda-data/documentation: modules/manage/partials/shadowing/shadow-link-prerequisites.adoc

// tag::cluster-property[]
The shadow cluster must have the xref:reference:properties/cluster-properties.adoc#enable_shadow_linking[`enable_shadow_linking`] cluster property set to `true`.

ifdef::env-cloud[]
[NOTE]
====

> Starting with Redpanda v25.3, this cluster property is enabled by default on new Redpanda Cloud clusters. Existing clusters created before v25.3 must enable this property manually. See xref:manage:cluster-maintenance/config-cluster.adoc[].

Reviewer: That's not really a problem. Whenever we have empty brackets, it just pulls the title from the original file. So it's actually better most of the time

> Starting with Redpanda v25.3, this cluster property is enabled by default on new Redpanda Cloud clusters. Existing clusters created before v25.3 must enable this property manually. See xref:manage:cluster-maintenance/config-cluster.adoc[].

Reviewer 2: ```suggestion
Starting with Redpanda v25.3, this cluster property is enabled by default on new Redpanda Cloud clusters. For existing clusters on versions earlier than v25.3, you must enable this property manually. See xref:manage:cluster-maintenance/config-cluster.adoc[].
```

endif::[]

ifndef::env-kubernetes[]

> ifndef::env-cloud[]

Reviewer 2: This block is still somehow displayed on the Cloud version on the page: https://deploy-preview-1549--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/disaster-recovery/shadowing/setup/#cluster-configuration I wonder if it's the double ifndefs.
